index.html

```diff
@@ -232,5 +232,42 @@ <h3>Skills</h3>
             </div>
         </div>
     </section>
+    <br>
+    <section>
+        <!-- Contact form -->
+        <div class="div4">
+            <h2 class="color1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+        <br>
+        <div class="div5">
+            <form id="contact-form" action="https://formspree.io/f/mlevonav" method="POST">
+                <div>
+                    <input class="input-fields" type="text" name="name" id="name" placeholder="Full Name" max="30" required>
+                </div>
+                <div>
+                    <input class="input-fields" type="email" name="email" id="email" placeholder="Email Address" required>
+                </div>
```

Comment:
- [ ] Kindly stick as much as possible to the design provided by the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) 🎨. At the moment, your contact section is missing some spacing:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/199342535-f65f6e04-9da4-449d-a7ca-e5e8b8b258c8.png) | ![imageedit_2_4750070421](https://user-images.githubusercontent.com/77069011/199342393-70f8b0e3-b350-40ec-86ed-5e88901c4932.gif) |

Please fix it! 🤓 🛠️